### DISCUSSION TITLE
How do I force cc65 to not translate my string?

### DISCUSSION THREAD

**Discussion by ReallyNiceGuy** [cc65/cc65#1453] [Q&A]
I am trying to use the ISO-8859-15 mode of cx16 target, but my characters keep being translated to PETSCII.
How do I disable it?

**Comment by mrdudz** [comment]
define the mapping accordingly using #pragma charmap ... i guess

**Reply by ReallyNiceGuy** [reply]
Yes, I think this is the way to go. I used a inc file with .charmap  x, x, from 1 to 255.
